Cargo.toml

```diff
@@ -27,13 +36,21 @@ fixed_window_roller = []
 size_trigger = []
 time_trigger = ["rand"]
 onstartup_trigger = []
-json_encoder = ["serde", "serde_json", "chrono", "log-mdc", "log/serde", "thread-id"]
+json_encoder = [
+    "serde",
+    "serde_json",
+    "chrono",
+    "log-mdc",
+    "log/serde",
+    "thread-id",
+]
 pattern_encoder = ["chrono", "log-mdc", "thread-id"]
 ansi_writer = []
 console_writer = ["ansi_writer", "libc", "winapi"]
 simple_writer = []
 threshold_filter = []
 background_rotation = []
+log_kv = ["log/kv"]
```

Comment:
Looks like log::kv is unstable in the current requested log version. Mind fixing this?